plugins/manual_classification/tasks.py

```diff
@@ -103,16 +110,17 @@ def add_class(self, db_id: int) -> str:
         TASK_LOGGER.error(msg)
         raise ValueError(msg)
 
-    entity_data = task_data.data["entity_data"]
-    for id in entity_data.keys():
+    entity_annotation = copy.deepcopy(task_data.data["entity_annotation"])
```

Comment:
I think it is possible to mark this as changed without a deep copy but I may be wrong. Either way this is something that deserves a warning in the documentation!

Reply:
The problem is that we use SQL Alchemy's JSON type in the key-value storage (for type of the value) that does not detect in-place mutations. Hence, the need of a deep copy. If you can think of a better solution, let me know. Else, I have noted it in the documentation.